src/routing/approved.rs

```diff
@@ -37,12 +37,18 @@ use crate::{
 use bls_dkg::key_gen::{message::Message as DkgMessage, outcome::Outcome as DkgOutcome};
 use bytes::Bytes;
 use itertools::Itertools;
+use lru_time_cache::LruCache;
+use resource_proof::ResourceProof;
 use std::{cmp, net::SocketAddr, slice};
 use tokio::sync::mpsc;
 use xor_name::{Prefix, XorName};
 
+pub(crate) const RESOURCE_PROOF_DATA_SIZE: usize = 256;
+pub(crate) const RESOURCE_PROOF_DIFF: u8 = 8;
```

Comment:
I would not abbreviate "difficulty" to "diff". It's confusing as "diff" typically means "difference".